src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -353,6 +352,25 @@ export default class AircraftController {
         return this.aircraft.models[icao];
     }
 
+    /**
+     * Remove the specified aircraft from `AircraftController.aircraft`
+     * @method removeAircraftInstanceModelFromList
+     * @param  {Aircraft} aircraft the aircraft to remove
+     */
+    removeAircraftInstanceModelFromList(aircraft) {
+        this.aircraft.list = _without(this.aircraft.list, aircraft);
+    }
+
+    /**
+     * Remove a flight number from the list stored in `AircraftController.aircraft.callsigns`
+     * @for AircraftController
+     * @method removeCallsignFromList
+     * @param  {string} callsign the flight number to remove
```

Comment:
moving in the same vein as `icao`, we use both `callsign` and `flight number` interchangeably it seems.  Which is the correct term?  callsign for a fully formed callsign and flight number for just the numbers?
```javascript
callsign = AA123
flightNumber = 123
```